### PR TITLE
Fix Channel Enumeration tutorial

### PR DESCRIPTION
**Pull request by marklewin**
The tutorial was broken and the live demo used Basic auth (see: [DOC-321](https://ably.atlassian.net/browse/DOC-321)). Also used the opportunity to rewrite for better clarity and replace an out of date screenshot.

### CODE REVIEW

**Review by m-hulbert — CHANGES_REQUESTED** [review]
Reads well to me! My only suggestion would be to replace the instances of "we" and "our" in the text. There's also one incorrect link to fix.

File: content/tutorials/channel-enumeration-rest.textile [hunk 1]
```diff
@@ -16,33 +16,49 @@ authors:
...
+# Ensure that Channel Metadata is selected in the list of capabilities
+# Ensure that Resource restrictions (channels & queues) is set to None
+# Click the Save button
+
+<a href="/images/tutorials/metadata/channel-metadata-capabilities.png" target="_blank">
+  <img src="/images/tutorials/metadata/channel-metadata-capabilities.png" style="width: 100%" alt="Channel metadata capabilities">
 </a>
 
-A regular Ably API key has a @capability@ which lists resources (glob expressions that match channel names) and, for any given resource, a set of permitted operations. The wildcard resource @'*'@ will match any regular channel name. To enable access to channel metadata, ensure the @Channel Metadata@ privilege is enabled in your API key.
+For simplicity, this tutorial will use "basic authentication":/core-features/authentication#basic-authentication. In production, "Ably advises that you use token authentication":http://localhost:4000/tutorials/channel-enumeration-rest/ in client-side applications to "protect your API key":/root/best-practice-guide#using-api-key-client-side.
```
[the remark this answers is not in the document]
Comment (marklewin) [reply content/tutorials/channel-enumeration-rest.textile:59]: Good spot!

**Comment by marklewin** [comment]
> Reads well to me! My only suggestion would be to replace the instances of "we" and "our" in the text. There's also one incorrect link to fix.

Couldn't find any "we" ... fixed a couple of "our"

**Review by tbedford — APPROVED** [review]
Checked under website and nanoc. Looks good. Also made sure live demo was working.

**Comment by m-hulbert** [comment]
> > Reads well to me! My only suggestion would be to replace the instances of "we" and "our" in the text. There's also one incorrect link to fix.
> 
> Couldn't find any "we" ... fixed a couple of "our"

There's just the one in `We provide a REST client library to make it straightforward to access the REST API...` and it's all good from me.

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**